services/sidecar/src/sidecar/celery_log_setup.py

```diff
@@ -19,14 +19,14 @@
 def setup_loggers(logger, *_args, **_kwargs):
     """ Customizes global loggers """
     for handler in logger.handlers:
```

Comment:
This is ok but we should consider using common log formats. 
In the case of aiohttp, for the access logs they follow this one: https://en.wikipedia.org/wiki/Common_Log_Format
Perhaps it would make more sense to have several well defined logs in each app instead of one per submodule.

Reply:
I fully agree to a comon log format. I just could not find it.